src/components/home/home-footer/home-footer.component.scss

```diff
@@ -0,0 +1,13 @@
+.footer-container {
+    background: #262a2e;
+    //height: 5vh;
+}
+
+.copyright-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: snow;
+    text-align: center;
+    font-size: 10px;
```

Comment:
Please use viewport units for text font size instead of pixel units because the text may blocked/ missing when resizing the browser.
![image](https://user-images.githubusercontent.com/76939137/120457491-139f9380-c3c9-11eb-96f8-e55515af897d.png)

Reply:
For the copyright text, 
strongly suggest it to overlay the image. 

<img width="1792" alt="Screenshot 2021-06-02 at 5 57 57 PM" src="https://user-images.githubusercontent.com/33477318/120461910-071d3a00-c3cd-11eb-814c-0178038b6892.png">


The view of having full vertical image is much pleasant from UX perspective.